docker/include/bin/start_faux

```diff
@@ -277,5 +277,9 @@ else
     echo Runtime config $conf_file not found.
 fi
 
+if [ -n "${options[wpa_supplicant]}" ]; then
```

Comment:
please use wpa like my original comment -- things become unwieldy very quickly with long names for these options.

Also note that you'll have to add that option in wherever you launch the faux container -- not sure if that's landed in a PR yet or not (doesn't need to be included here -- that's just FYI).

Reply:
Changed.